app/controllers/Application.scala

```diff
@@ -151,7 +151,8 @@ object Application extends Controller {
     val existing = for {
       path <- notebookPath
       (id, kernel) <- KernelManager.atPath(path)
-    } yield (id, kernel, kernelIdToCalcService(id))
```

Comment:
`kernelIdToCalcService(id)` was not always existing (e.g. kernel already closed), so it raised a `NoSuchElementException`